includes/classes/pmpro-testimonial-form.php

```diff
@@ -98,21 +99,18 @@ public function display( $echo = true ) {
 									<h2 class="<?php echo esc_attr( pmpro_get_element_class( 'pmpro_form_heading pmpro_font-large' ) ); ?>"><?php esc_html_e( 'Submit Your Testimonial', 'pmpro-testimonials' ); ?></h2>
 								</legend>
 								<div class="<?php echo esc_attr( pmpro_get_element_class( 'pmpro_form_fields' ) ); ?>">
-									<?php
-									// Testimonial field
-									$title_field = new PMPro_Field(
-										'testimonial',
-										'textarea',
-										array(
-											'label'        => __( 'Testimonial', 'pmpro-testimonials' ),
-											'required'     => true,
-											'showrequired' => 'label',
-										)
-									);
-									$title_field->displayAtCheckout();
+									
+									<div class="<?php echo esc_attr( pmpro_get_element_class( 'pmpro_form_field pmpro_form_field-required pmpro_form_field-textarea', 'testimonial' ) ); ?>">
```

Comment:
This adds the `pmpro_form_field-required` class, but doesn't include a check for whether it is required. We could probably avoid adding that selector here, only because it isn't used by JS (like core PMPro) to show contextual "errors" on form submit.